dispatch/apps/events/templates/submit.html

```diff
@@ -0,0 +1,157 @@
+
+<script src="https://code.jquery.com/jquery-2.1.4.min.js" type="text/javascript"></script>
+<link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/ubyssey/static/css/jquery.datetimepicker.min.css" />
+<script type="text/javascript" src="https://s3-us-west-1.amazonaws.com/ubyssey/static/js/jquery.datetimepicker.full.min.js"></script>
+
+{{ form.non_field_errors }}
+<form method="post" enctype="multipart/form-data">
+  {% csrf_token %}
+
+  <div class="fieldWrapper">
+    {{ form.title.errors }}
+    <label for="{{ form.title.id_for_label }}">Title of Event:</label>
+    {{ form.title }}
+  </div>
+
+  <div class="fieldWrapper">
+    {{ form.description.errors }}
+    <label for="{{ form.description.id_for_label }}">Description:</label>
+    {{ form.description }}
+  </div>
+
+  <div class="fieldWrapper">
+    {{ form.host.errors }}
+    <label for="{{ form.host.id_for_label }}">Host:</label>
+    {{ form.host }}
+  </div>
+
+  {% if form.facebook_image_url.value %}
+  <div class="fieldWrapper">
+
+    {{ form.facebook_image_url.errors }}
+
+    <label for="{{ form.facebook_image_url.id_for_label }}">Image for the Event:</label>
+
+    <img id="previewImage" src="{{ form.facebook_image_url.value }}" style="-webkit-user-select: none; display: block; margin: auto; cursor: zoom-in;" >
+
+    {{ form.image.errors }}
+    <label for="{{ form.image.id_for_label }}">Or choose your own image:</label>
+    <input id="imgInp" type='file' onchange="readUrl(this);" name="image"/>
+
+    <input id="reset" type=reset onclick="
+    document.getElementById('previewImage').src='{{ form.facebook_image_url.value }}';
+    document.getElementById('reset').style.display = 'none';"
+    value="Or stick with the facebook image" />
+
+  </div>
+
+  {% else %}
+  <div class="fieldWrapper">
+    {{ form.image.errors }}
+    <label for="{{ form.image.id_for_label }}">Image for the Event:</label>
+    <input id="imgInp" type='file' onchange="readUrl(this);" name="image"/>
+    <img id="previewImage" src="#" alt="User uploaded image"/>
+  </div>
+  {% endif %}
+
+  <div class="fieldWrapper">
+    {{ form.start_time.errors }}
+    <label for="{{ form.start_time.id_for_label }}">Start Time:</label>
+    {{ form.start_time }}
+  </div>
+
+  <div class="fieldWrapper">
+    {{ form.end_time.errors }}
+    <label for="{{ form.end_time.id_for_label }}">End Time:</label>
+    {{ form.end_time }}
+  </div>
+
+  <div class="fieldWrapper">
+    {{ form.location.errors }}
+    <label for="{{ form.location.id_for_label }}">Location:</label>
+    {{ form.location }}
+  </div>
```

Comment:
just letting you know that you could also modify `DATETIME_INPUT_FORMATS` in the local `settings.py` (eg. `dispatch/conf/default_settings.py`) so that you don't have to edit the django dist files

I think the datetimepicker takes a format argument as well. (might be more robust to configure it to return an ISO date string)
http://xdsoft.net/jqplugins/datetimepicker/#use_other_date_parser